public/components/health-check/lib/check-time-filter.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosResponse } from 'axios';
+import { GenericRequest } from '../../../react-services';
+
+type userValue<T> = {userValue: T}
+
+type kbnSettings = {
+  buildNum: userValue<number>
+  timeFilter?: userValue<string[]>,
+};
+type responseKbnSettings = {settings: kbnSettings};
+
+const timeFilterSetting = [
+    { from: "nowtime-24h" },
+    { to: 'now'}];
+
+export function checkKibanaSettingsTimeFilter (removeMetaFields: boolean) {
+  removeMetaFields && getKibanaSettings()
```

Comment:
maybe when the user disables this option and the prev state was true should remove the userValue, but be careful if the setting is set from another way